src/main/kotlin/ar/edu/unahur/obj2/vendedores/CentroDeDistribucion.kt

```diff
@@ -0,0 +1,20 @@
+package ar.edu.unahur.obj2.vendedores
+
+class CentroDeDistribucion(val ciudadDeOrigen: Ciudad, val vendedores: MutableList<Vendedor>) {
+
+    fun agregarUnVendedor(vendedor: Vendedor) {
+        if (vendedores.contains(vendedor)) {
+            error("El vendedor ya está registrado en el centro de distribución.")
+        }
+        vendedores.add(vendedor)
+    }
+
+    fun vendedorEstrella() = vendedores.maxBy { it.puntajeCertificaciones() }
+
+    fun puedeCubrirUnaCiudad(ciudad: Ciudad) = vendedores.any() {it.puedeTrabajarEn(ciudad)}
```

Comment:
Ojo, que los paréntesis estos hacen que no se ejecute el bloque que viene después, y va a dar `true` si la lista tiene algún elemento, no importa cuál. Sería así:

```suggestion
    fun puedeCubrirUnaCiudad(ciudad: Ciudad) = vendedores.any {it.puedeTrabajarEn(ciudad)}
```